tests/test_examples.py

```diff
@@ -24,7 +24,21 @@ class TestExamples(unittest.TestCase):
     loader = importlib.machinery.SourceFileLoader("example", example)
     mod = types.ModuleType(loader.name)
     loader.exec_module(mod)
-    
+
+    # run again with gurobi
+    if "npuzzle" in example or "tst_likevrp" in example:
```

Comment:
We should make this better/modular
i.e. a list of solver names to run the tests on at the top of the file